src/resolvers/Topics.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Promise = require('bluebird');
+const blobStorageManager = require('../storageClients/BlobStorageManager');
+
+module.exports = {
+  get(args) {
```

Comment:
I am going to create a wrapper for appinsights that we can use so that we can pull out logging from the business logic.